packages/host/app/components/operator-mode/code-submode/left-panel-toggle.gts

```diff
@@ -3,6 +3,9 @@ import { on } from '@ember/modifier';
 import { inject as service } from '@ember/service';
 import Component from '@glimmer/component';
 
+import FileCheck from '@cardstack/boxel-icons/file-check';
+import FolderTree from '@cardstack/boxel-icons/folder-tree';
```

Comment:
unfortunately we're gonna have to copy these icons to `boxel-ui/icons` and import from there for host

Reply:
From our research it does seem like this is okay! So I’ll resolve this.